From 6ad5fa5c5ef098a2d5a62d3cd19886ce8501fbcf Mon Sep 17 00:00:00 2001
From: Wayne Zou <b36644@freescale.com>
Date: Wed, 28 Dec 2011 15:01:25 +0800
Subject: [PATCH 262/505] ENGR00171016 fb unit test: use fb0 and fb1 for fb
 test

fb unit test: use fb0 as BG and fb1 as FG on the same IPU for fb test.
fb0 and fb1 should be on the same display device.

Signed-off-by: Wayne Zou <b36644@freescale.com>
---
 test/mxc_fb_test/mxc_fb_test.c | 11 +----------
 1 file changed, 1 insertion(+), 10 deletions(-)

diff --git a/test/mxc_fb_test/mxc_fb_test.c b/test/mxc_fb_test/mxc_fb_test.c
index afe335d..17d315e 100644
--- a/test/mxc_fb_test/mxc_fb_test.c
+++ b/test/mxc_fb_test/mxc_fb_test.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2009 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2004-2011 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -296,15 +296,6 @@ main(int argc, char **argv)
         {
                 goto err1;
         }
-	if (strcmp(fb_fix.id, "DISP3 FG") != 0) {
-		close(fd_fb1);
-		if ((fd_fb1 = open("/dev/fb2", O_RDWR, 0)) < 0)
-		{
-			printf("Unable to open /dev/fb2\n");
-			retval = TFAIL;
-			goto err0;
-		}
-	}
         retval = ioctl(fd_fb1, FBIOBLANK, FB_BLANK_UNBLANK);
         if (retval < 0)
         {
-- 
1.8.0

